src/Compilers/Core/Portable/Compilation.EmitStream.cs

```diff
@@ -42,11 +42,16 @@ internal enum EmitStreamSignKind
         /// a temporary file must be created on disk (at the last possible moment), emitted to, signed on disk
         /// and then copied back to the original <see cref="Stream"/>. Only when legacy signing is enabled though.
         /// </summary>
+        /// <remarks>
+        /// The <see cref="Stream"/> returned here should not be disposed by the caller. It is either owned by 
```

Comment:
I find this language a little confusing. A type doesn't return anything. Was this remark meant to be on 'CreateStream'?

Reply:
Moved the comment to the more appropriate location.